packages/core/email/.eslintrc.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     },
     {
       files: ['**/*'],
-      excludedFiles: ['admin/**/*'],
```

Comment:
Why do you have to exclude the server part? I think this shouldn't be necessary, because this applies to server-code only.